setup.py

```diff
@@ -38,6 +38,9 @@
     if os.path.exists(pjoin(d, '__init__.py')):
         packages.append(d[len(here)+1:].replace(os.path.sep, '.'))
 
+package_data = {'nb_conda_kernels': [pjoin('logos', 'python', '*'),
```

Comment:
Right, but the kernel has to _be there_ in order for us to use it. If it is installed, if follows, it also has it's resource data:
```bash
~/miniconda/envs/nb_conda_kernels$ find . | grep 64x64
./lib/python3.5/site-packages/ipykernel/resources/logo-64x64.png
./lib/R/library/IRkernel/kernelspec/logo-64x64.png
./share/jupyter/kernels/ir/logo-64x64.png
```
Unfortunately, there's no reproducible way to figure that out, i'd imagine :(